docs/content/users/quickstart.md

```diff
@@ -737,27 +737,14 @@ ddev launch /admin
 
 There are many ways to install Symfony, here are a few of them based on the [Symfony docs](https://symfony.com/doc/current/setup.html).
 
-If your project uses a database you'll want to set the [DB connection string](https://symfony.com/doc/current/doctrine.html#configuring-the-database) in the `.env`. If using the default MariaDB configuration, you'll want `DATABASE_URL="mysql://db:db@db:3306/db?serverVersion=10.11"`. If you're using a different database type or version, see `ddev describe` for the type and version.
-
-=== "Composer"
-
-    ```bash
-    mkdir my-symfony-site && cd my-symfony-site
-    ddev config --docroot=public
-    ddev composer create symfony/skeleton
-    ddev composer require webapp
-    # When it asks if you want to include docker configuration, say "no" with "x"
-    ddev launch
-    ```
```

Comment:
Why is this removed?

Reply:
even the exsiting guide using Symfony CLI also does a tricks by using a tmp dir  :stuck_out_tongue:

> I don't think it should be removed.
yeah, we can revert it